### PR TITLE
#130 - Terrain sorting

### LINKED ISSUE #130: TODO - Terrain Sorting (ALTIS_TANOA macro)
Improve the terrain sorting system, as the current macro is very restrictive. Also allows an easier switch to different terrains, re the recent 'mod' changes.

### PR DESCRIPTION
**Pull request by BoGuu**
Resolves #130

#### Changes proposed in this pull request: 
See #130 

Need everyone to have a quick check through this. Also, untested, can't access ARMA currently.

### CODE REVIEW

**Comment by SimZor** [comment]
Totally works!

File: Altis_Life.Altis/core/functions/fn_terrainSort.sqf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*
+    File: fn_terrainSort.sqf (TO BE CHANGED)
+    Author: BoGuu
+
+    Description:
+    Gah.
```
Comment (tkcjesse) [inline Altis_Life.Altis/core/functions/fn_terrainSort.sqf]: Hi Tonic.

**Comment by Jawshy** [comment]
Merge conflict. :cry:

**Comment by SimZor** [comment]
It's in ```fn_jailMe.sqf``` @Jawshy 

Shouldn't be that hard to fix, just like 4 lines of code.

**Pull request merged**